internal/clients/helpers.go

```diff
@@ -132,3 +133,17 @@ func toStringMap(m types.Map) map[string]string {
 
 	return result
 }
+
+func managedBy() (string, string) {
+	const (
+		empty       = ""
+		byTask      = "task"
+		env         = "TASK_UUID"
+		byTerraform = "terraform"
+	)
+	if taskId := os.Getenv(env); len(taskId) >= 1 {
```

Comment:
where do you get the env var from?

Reply:
in Terraform Cloud workspace i can set env vars.